src/graspitGUI.cpp

```diff
@@ -157,6 +157,15 @@ GraspItGUI::processArgs(int argc, char** argv)
         return FAILURE;
     }
 
+#ifdef CGDB_ENABLED
+    //Ensure $CGDB_MODEL_ROOT is set.
+    QString modelRoot = QString(getenv("CGDB_MODEL_ROOT"));
+    if (modelRoot.isNull() ) {
+      std::cerr << "You've built Graspit with the CGDB enabled. Please set CGDB_MODEL_ROOT or disable CGDB." << std::endl;
+      return FAILURE;
+    }
+#endif
+
```

Comment:
At this point, the CGDB_ENABLED is being used to really mean "DATABASE_ENABLED" which could mean for example the ROS household_objects_database. This one does not use the CGDB_MODEL_ROOT environment variable, so I would not add this.